proof/deserialize-p1.saw

```diff
@@ -323,7 +319,7 @@ custom_prove_cryptol
 
 add_fp_noabs_ov <- admit "add_fp" add_fp_noabs_spec;
 
-sgn0_pty_mont_384_ov <- admit "sgn0x_pty_mont_384" sgn0_pty_mont_384_spec;
+sgn0_pty_mont_384_ov <- admit "sgn0_pty_mont_384" sgn0_pty_mont_384_spec;
```

Comment:
This depends on whether __ADX__ is defined, and thus can lead to an error depending on the machine used to check the proof.